ivy_tests/test_ivy/helpers.py

```diff
@@ -1803,37 +1807,46 @@ def array_values(
     values = None
     if "int" in dtype:
         if dtype == "int8":
-            min_value = ivy.default(min_value, round(-128 * safety_factor))
-            max_value = ivy.default(max_value, round(127 * safety_factor))
+            min_value = ivy.default(min_value, round(-128 * large_value_safety_factor))
+            max_value = ivy.default(max_value, round(127 * large_value_safety_factor))
         elif dtype == "int16":
-            min_value = ivy.default(min_value, round(-32768 * safety_factor))
-            max_value = ivy.default(max_value, round(32767 * safety_factor))
+            min_value = ivy.default(
+                min_value, round(-32768 * large_value_safety_factor)
+            )
+            max_value = ivy.default(max_value, round(32767 * large_value_safety_factor))
         elif dtype == "int32":
-            min_value = ivy.default(min_value, round(-2147483648 * safety_factor))
-            max_value = ivy.default(max_value, round(2147483647 * safety_factor))
+            min_value = ivy.default(
+                min_value, round(-2147483648 * large_value_safety_factor)
+            )
+            max_value = ivy.default(
+                max_value, round(2147483647 * large_value_safety_factor)
+            )
         elif dtype == "int64":
             min_value = ivy.default(
-                min_value, round(-9223372036854775808 * safety_factor)
+                min_value, round(-9223372036854775808 * large_value_safety_factor)
             )
             max_value = ivy.default(
-                max_value, round(9223372036854775807 * safety_factor)
+                max_value, round(9223372036854775807 * large_value_safety_factor)
             )
         elif dtype == "uint8":
-            min_value = ivy.default(min_value, round(0 * safety_factor))
-            max_value = ivy.default(max_value, round(255 * safety_factor))
+            min_value = ivy.default(min_value, round(0 * large_value_safety_factor))
```

Comment:
just realized, the `large_value_safety_factor` is not relevant for `min_value` of unsigned integers, so we should remove this for all of them.